clientv3/lease.go

```diff
@@ -44,6 +44,22 @@ type LeaseKeepAliveResponse struct {
 	TTL int64
 }
 
+// LeaseTimeToLiveResponse is used to convert the protobuf lease timetolive response.
+type LeaseTimeToLiveResponse struct {
+	*pb.ResponseHeader
+	ID LeaseID `json:"id"`
+
+	// TTL is the remaining TTL in seconds for the lease.
+	// Milliseconds will be rounded off.
```

Comment:
`TTL is the remaining TTL in seconds for the lease; the lease will expire in under TTL+1 seconds.`?